examples/everything.amp.html

```diff
@@ -318,6 +341,14 @@ <h1 id="top">AMP #0</h1>
   <div id="footer">The end. <a href="#top">Back to top.</a></div>
 
   <amp-pixel src="https://pubads.g.doubleclick.net/activity;dc_iu=/12344/pixel;ord=$RANDOM?"></amp-pixel>
+  <amp-font
+    layout="nodisplay"
+    font-family="Source Sans Pro"
+    timeout="3000"
+    while-loading-class="sanspro-font-loading"
+    on-error-class="sanspro-font-missing"
+    on-load-class="sanspro-font-loaded">
```

Comment:
No - that is why I added the while-loading class - I dont think we would need a remove class.